### PR TITLE
Use shell scripts for Travis

### LINKED ISSUE #59: The travis file is getting ugly
Some thoughts:

- YML is not the best language to write commands, shell is (because of syntactic coloration and things like [ShellCheck](https://github.com/koalaman/shellcheck)).
- Only Travis can run the commands, if you want to reproduce a failed build locally, you have to do some copy pasting.
- If me move to another CI solution, some things will change but some things should not.

In the end it would be great to do something like this :

```yml
before_install:
   - source scripts/ci.sh
   - before_install # call a function defined in ci.sh
```

### PR DESCRIPTION
**Pull request by Soullivaneuh**
Fixes #59.

The goal is to have a nicer view of the configured scripts.

Travis build: https://travis-ci.org/sonata-project/exporter/builds/131681135

### CODE REVIEW

File: project/.travis/before_install_docs.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env sh
+set -ev
```
Comment (greg0ire) [inline project/.travis/before_install_docs.sh]: Maybe check if the file exists before executing it instead?

Comment (Soullivaneuh) [inline project/.travis/before_install_docs.sh]: I prefer keep this file even if is empty.

With this, people know that those files are managed by dev-kit and not by the project.

File: project/.travis/install_test.sh [hunk 2]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env sh
+set -ev
+
+mkdir --parent "${HOME}/bin"
```
Comment (greg0ire) [inline project/.travis/install_test.sh]: `parent` => `parents`

File: project/.travis/install_test.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+# PHPUnit install
+if [ ${TRAVIS_PHP_VERSION} '<' '5.6' ]; then
+    wget https://phar.phpunit.de/phpunit-old.phar --output-document="${HOME}/bin/phpunit"
+else
+    wget https://phar.phpunit.de/phpunit.phar --output-document="${HOME}/bin/phpunit"
+fi
```
Comment (greg0ire) [inline project/.travis/install_test.sh]:
```sh
if [ ${TRAVIS_PHP_VERSION} '<' '5.6' ]; then
    PHPUNIT_PHAR=phpunit-old.phar
else
    PHPUNIT_PHAR=phpunit.phar
fi
wget "https://phar.phpunit.de/${PHPUNIT_PHAR}" --output-document="${HOME}/bin/phpunit"
```

**Comment by Soullivaneuh** [comment]
@greg0ire Are we ok now ?

**Comment by greg0ire** [comment]
Perfect :+1:

**Pull request merged**